src/xrc/xh_menu.cpp

```diff
@@ -101,6 +114,17 @@ wxObject *wxMenuXmlHandler::DoCreateResource()
             wxMenuItem *mitem = new wxMenuItem(p_menu, id, label,
                                                GetText(wxT("help")), kind);
 #if wxUSE_ACCEL
+            if (!extra_accels.empty())
+            {
+                const int entries_size = extra_accels.size();
+                for (wxString extra_accel : extra_accels)
```

Comment:
Unfortunately we still can't use C++11 in wx 3.1, please rewrite the loop using indices (and then we'll rewrite it back like this after 3.2.0...).